components/SearchBar.tsx

```diff
@@ -20,34 +21,44 @@ const SearchBar = ({
   onSubmit,
   defaultValue,
   placeholder,
+  maxWidth = '276px',
```

Comment:
Can we not enforce the default for this one and rather set it on the pages that require it? Just trying to prevent other regressions.

```suggestion
  maxWidth = '100%',
```